app/_hub/kong-inc/upstream-timeout/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Upstream Timeout
+publisher: Kong Inc.
+categories:
+  - traffic-control
+type: plugin
+
+desc: Set timeouts on routes and override service-level timeouts
+description: |
+  Use the Upstream Timeout plugin to configure route-specific timeouts.
+  This plugin overrides any service-level timeout settings.
+
+kong_version_compatibility:
+  enterprise_edition:
+    compatible: true
+
+cloud: true
+enterprise: true
+plus: false
+
+params:
+  name: upstream-timeout
+  service_id: false
+  consumer_id: false
+  route_id: true
+  protocols:
+    - http
+    - https
+  dbless_compatible: yes
+  config:
+    - name: connect_timeout
+      required: false
+      default: null
+      value_in_examples: 4000
+      datatype: integer
+      description: |
+        The timeout, in milliseconds, for establishing a connection to the upstream server.
+        Overrides the [service object `connect_timeout` setting](/gateway/latest/admin-api/#service-object).
```

Comment:
```suggestion
        Overrides the service object [`connect_timeout`](/gateway/latest/admin-api/#service-object) setting .
```

Reply:
Would this be a better link here? 
https://deploy-preview-4402--kongdocs.netlify.app/gateway/3.0.x/how-kong-works/routing-traffic/#3-proxying-and-upstream-timeouts 

since it defines what that setting is?

Reply:
I didn't even know we had that! Yes, that works.